src/lib/apollo/client.ts

```diff
@@ -34,7 +34,20 @@ const scalarsLink = withScalars({ schema, typesMap })
 const httpLink = new HttpLink({ uri: subgraphUri() })
 
 const client = new ApolloClient({
-  cache: new InMemoryCache(),
+  cache: new InMemoryCache({
+    typePolicies: {
+      Query: {
+        fields: {
+          projectEvents: {
+            keyArgs: [],
+            merge(existing = [], incoming) {
```

Comment:
necessary for `fetchMore` functionality. maybe more work to be done here

https://www.apollographql.com/docs/react/pagination/core-api